bin/hdfcoinc/pycbc_coinc_statmap

```diff
@@ -19,111 +22,186 @@ class fw(object):
         self.attrs = self.f.attrs
 
     def __setitem__(self, name, data):
-        self.f.create_dataset(name, data=data, compression='gzip',
-                              compression_opts=9, shuffle=True)
+        # Make a new item if isn't in the hdf file
+        if not name in self.f:
+            self.f.create_dataset(name, data=data, compression="gzip",
+                                  compression_opts=9, shuffle=True,
+                                  maxshape=data.shape)
+        # Else re-size that section and reassign values
```

Comment:
@stevereyes01 In a later comment, I request deleting that section, so you shouldn't need it.